### PR TITLE
feat: #75 make camera controls option configurable and add prop types…

### PR DESCRIPTION
**Pull request by paularah**

### CODE REVIEW

File: geppetto.js/geppetto-ui/src/3d-canvas/Canvas.js [hunk 1]
```diff
@@ -294,7 +306,42 @@ Canvas.propTypes = {
   /**
    * Options to customize camera
    */
-  cameraOptions: PropTypes.object,
+  cameraOptions: PropTypes.shape({
```
Comment (ddelpiano) [inline geppetto.js/geppetto-ui/src/3d-canvas/Canvas.js:309]: well done @paularah , this will save time to all to see what the props are! 👍🏽

Comment (ddelpiano) [reply geppetto.js/geppetto-ui/src/3d-canvas/Canvas.js:309]: one small note, the propTypes are currently missing zoomSpeed and panSpeed, rotateSpeed instead is there. Thx

**Review by ddelpiano — CHANGES_REQUESTED** [review]

**Comment by paularah** [comment]
@enicolasgomez essentially what's happening with the merged and default props is that deeply nested default props aren't merged by react. If fields like incrementPan & IncrementRotation aren't explicitly passed as props their values result to undefined despite being specified in the default props of the component. A possible workaround is to keep both the merged props, component default props and default props object  then let the spread operator fill in the fields

**Review by ddelpiano — APPROVED** [review]

**Comment by ddelpiano** [comment]
@paularah could you please fix the conflict here? 
if you need any help let us know, thx

**Pull request merged**